views/js/helper/status.js

```diff
@@ -69,8 +79,8 @@ define(['lodash', 'i18n'], function(_, __){
                 report : true
             }
         },
-        terminated : {
-            code : 'TERMINATED',
+       FINISHED : {
+            code : 'FINISHED',
```

Comment:
that should remain "terminated" and "TERMINATED", the  initial mistake was on the server side sorru